mail/alpine/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 
 name                alpine
 version             2.25
-revision            2
+revision            3
```

Comment:
I don't think there is a reason to increase the `revision` field here

Reply:
You are correct here as well.  Haven't done any macports or BSD ports files in a while so I forgot the rule on revision.